internal/terraform/module/module_ops_test.go

```diff
@@ -1286,9 +1366,14 @@ func TestSchemaVarsValidation_SingleFile(t *testing.T) {
 	}
 
 	fs := filesystem.NewFilesystem(ss.DocumentStore)
+	fooURI, err := filepath.Abs(filepath.Join(modPath, "terraform.tfvars"))
+	if err != nil {
+		t.Fatal(err)
+	}
 	ctx = lsctx.WithRPCContext(ctx, lsctx.RPCContextData{
 		Method: "textDocument/didChange",
-		URI:    "file:///test/terraform.tfvars",
+		URI:    uri.FromPath(fooURI),
+		// URI:    "file:///test/terraform.tfvars",
```

Comment:
The ParseVariableFile function is expecting a fully qualified URI here. Is this OK?

Reply:
The above (not the commented out code) seems reasonable to me.